src/db/sysdb_search.c

```diff
@@ -675,7 +677,13 @@ errno_t sysdb_search_with_ts_attr(TALLOC_CTX *mem_ctx,
         }
         ts_cache_res->count = 0;
         ts_cache_res->msgs = NULL;
-    } else {
+
+        break;
+    }
+
+    case SYSDB_CACHE_TYPE_TIMESTAMP:
```

Comment:
We use an explicit `/* FALLTHOUGH*/` comment in SSSD in case one switch-case falls through to another. It helps visually, but also helps static code analyzers.

Reply:
So replace empty spaces between cases by /* FALLTHROUGH */ line?

Reply:
Actually, sorry. What we rather use is SSS_ATTRIBUTE_FALLTHROUGH. Feel free to grep for its usage and definition in configure.ac.

Reply:
Done